src/scene/graphics/quad-render-utils.js

```diff
@@ -81,14 +81,14 @@ function drawQuadWithShader(device, target, shader, rect, scissorRect) {
 /**
  * Draws a texture in screen-space. Mostly used by post-effects.
  *
- * @param {import('../../platform/graphics/graphics-device.js').GraphicsDevice} device - The graphics device used to draw
+ * @param {import('../../platform/graphics/webgl/webgl-graphics-device.js').WebglGraphicsDevice} device - The graphics device used to draw
  * the texture.
  * @param {import('../../platform/graphics/texture.js').Texture} texture - The source texture to be drawn. Accessible as
  * `uniform sampler2D * source` in shader.
  * @param {import('../../platform/graphics/render-target.js').RenderTarget} [target] - The destination render target.
  * Defaults to the frame buffer.
  * @param {import('../../platform/graphics/shader.js').Shader} [shader] - The shader used for rendering the texture.
- * Defaults to {@link GraphicsDevice#getCopyShader}.
+ * Defaults to {@link WebglGraphicsDevice#getCopyShader}.
```

Comment:
An alternative here is to simply remove the `Defaults to...` sentence and keep `getCopyShader` out of the docs. @mvaligursky?

Reply:
Reference removed in 44a127d
`getCopyShader` set back to `@ignore` in ef35b41